.github/workflows/node.js.yml

```diff
@@ -0,0 +1,30 @@
+name: Angular build
+
+on:
+  push:
+    branches: [ 'main' ]
+  pull_request:
+    branches: [ 'main' ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x, 22.x]
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+        cache-dependency-path: './framezzz-frontend/package-lock.json'
+    - name: Install dependencies
+      working-directory: ./framezzz-frontend
+      run: npm i
+    - name: Build Angular app
+      working-directory: ./framezzz-frontend
+      run: npm run build
```

Comment:
I think it also worth to implement here running tests since it's already implemented and ready for integration

Reply:
Done 👍